test/showcase/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+load(
+    "//rules_java_gapic:java_gapic.bzl",
+    "java_gapic_library",
+    "java_gapic_test",
+)
+load("@io_grpc_grpc_java//:java_grpc_library.bzl", "java_grpc_library")
+load("@rules_gapic//:gapic.bzl", "proto_library_with_info")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//rules_java_gapic:java_gapic_pkg.bzl", "java_gapic_assembly_gradle_pkg")
+
+package(default_visibility = ["//visibility:public"])
+
+proto_library_with_info(
+    name = "showcase_proto_with_info",
+    deps = [
+        "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto",
+        "@com_google_googleapis//google/cloud:common_resources_proto",
+    ],
+)
+
+java_proto_library(
+    name = "showcase_java_proto",
+    deps = ["@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto"],
+)
+
+java_grpc_library(
+    name = "showcase_java_grpc",
+    srcs = ["@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_proto"],
+    deps = [":showcase_java_proto"],
+)
+
+java_gapic_library(
+    name = "showcase_java_gapic",
+    srcs = [":showcase_proto_with_info"],
+    gapic_yaml = None,
+    grpc_service_config = "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_grpc_service_config.json",
+    rest_numeric_enums = True,
+    service_yaml = "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_v1beta1.yaml",
+    test_deps = [
+        ":showcase_java_grpc",
+    ],
+    transport = "grpc+rest",
+    deps = [
+        ":showcase_java_proto",
+        "@com_google_googleapis//google/api:api_java_proto",
+    ],
+)
+
+java_gapic_test(
```

Comment:
Are these tests run with `bazel --batch build //test/showcase:all`?

Reply:
They are invoked with `bazel test //test/showcase:all`, not `build`.

Reply:
I don't see `bazel test //test/showcase:all` in this PR, that's why I'm wondering if this is triggered or not at this point. There is an identical `java_gapic_test` in the Bazel build file in integration folder, which I think will be triggered by `bazel test //...` automatically, but is this one invoked or not?
The bottom line is I think we only want to have these target declared and invoked in one place to prevent confusion.

Reply:
Yes - makes sense and I agree. I've removed the test suite from /test/showcase so those can focus on the actual integration tests w/ the server.